awscodecommit/main.go

```diff
@@ -259,17 +265,28 @@ func (clients Clients) preparePullRequests() ([]*codecommit.PullRequest, error)
 
 //sendPush sends an event containing data about a git commit that was pushed to a branch
 func (clients Clients) sendCommitEvent(commit *codecommit.Commit) error {
+
 	log.Info("send Commit Event")
 
-	codecommitEvent := PushMessageEvent{
-		Commit:           commit,
-		CommitRepository: aws.String(repoNameEnv),
-		CommitBranch:     aws.String(repoBranchEnv),
-		EventSource:      aws.String("aws:codecommit"),
-		AwsRegion:        aws.String(awsRegionEnv),
+	event := cloudevents.Event{
+		Context: cloudevents.EventContextV03{
+			Type:            "AWS CodeCommit commit event",
+			Subject:         aws.String("AWS CodeCommit Event"),
```

Comment:
`subject`, if present, should be more descriptive. It needs to be a unique descriptor (URL) of the entity upon which the event happened. For example, perhaps the repo, or branch, depending on the type of event.

Reply:
this applies to the rest of the PR - so I won't repeat myself